### PR TITLE
update logger and LJM to support log timestamps

### PR DESCRIPTION
**Pull request by paweldomas**
Depends on https://github.com/jitsi/lib-jitsi-meet/pull/991 and https://github.com/jitsi/jitsi-meet-logger/pull/6

### CODE REVIEW

File: react/features/base/logging/JitsiMeetLogStorage.js [hunk 1]
```diff
@@ -63,13 +63,13 @@ export default class JitsiMeetLogStorage {
         for (let i = 0, len = logEntries.length; i < len; i++) {
             const logEntry = logEntries[i];
 
-            if (typeof logEntry === 'object') {
-                // Aggregated message
-                logMessage += `(${logEntry.count}) ${logEntry.text}\n`;
-            } else {
-                // Regular message
-                logMessage += `${logEntry}\n`;
+            if (logEntry.timestamp) {
```
Comment (zbettenbuk) [inline react/features/base/logging/JitsiMeetLogStorage.js]: What happens if logEntry is undefined?

Comment (paweldomas) [reply react/features/base/logging/JitsiMeetLogStorage.js]: it is never supposed to be undefined, the app will crash

Comment (paweldomas) [reply react/features/base/logging/JitsiMeetLogStorage.js]: I think that even if log method is called empty, then the logger lib will still produce the logEntry object with a timestamp but no text. I will check if that cases works. I can't think of any other case that could lead to problems here.

Comment (paweldomas) [reply react/features/base/logging/JitsiMeetLogStorage.js]: Yeah just tested the empty case. I don't think this should ever crash unless the logger library is broken which should be detected on the dev stage.

**Comment by paweldomas** [comment]
Yes, it's been tested on web, android and ios and looks good to me.

**Pull request merged**